Lib/fontTools/misc/loggingTools.py

```diff
@@ -463,6 +463,13 @@ def emit(self, record):
 	def createLock(self):
 		self.lock = None
 
+	def match(self, regexp):
```

Comment:
Maybe call the method "assertMatch", to make clear that it will raise an error if there’s no match?